packages/legend-application-studio/src/stores/graphModifier/StoreRelational_GraphModifierHelper.ts

```diff
@@ -421,3 +431,78 @@ export const rootRelationalSetImp_setPropertyMappings = action(
     );
   },
 );
+
+// --------------------------------------------- Post-Processor -------------------------------------
+
+export const relationalDatabaseConnection_addNewMapperPostProcessor = action(
+  (
+    connectionValueState: RelationalDatabaseConnectionValueState,
+    observeContext: ObserverContext,
+  ): void => {
+    addUniqueEntry(
+      connectionValueState.connection.postProcessors,
+      observe_PostProcessor(new MapperPostProcessor(), observeContext),
+    );
+  },
+);
+
+export const postProcessor_addMapper = action(
```

Comment:
--> `mapperPostProcessor_addMapper` and pass the new mapper in (be it schema mapper or normal mapper)